apps/sms/test/marionette/lib/views/conversation/accessors.js

```diff
@@ -110,6 +110,16 @@ ConversationAccessor.prototype = {
     this.actions.flick(conversationContainer, 50, 50, 50, 350).perform();
   },
 
+  fakeScrollUpTo: function(scrollTop) {
+    var conversationContainer = this.client.findElement(
+      SELECTORS.container
+    );
+
+    conversationContainer.scriptWith(function(container, scrollTop) {
+      container.scrollTop = scrollTop;
+    }, scrollTop);
+  },
```

Comment:
well with scrollUp you don't really control well what's happening. I only fear that sometimes we could not scroll enough to actually get something displayed (for example if I change the chunk size to 50)... But I admit I didn't try at all with the scrollUp function so I might just try already.